### PR TITLE
Update comments for installations, performance and remote config

### PR DESCRIPTION
**Pull request by Feiyang1**

### CODE REVIEW

File: packages-exp/functions-exp/src/index.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/**
+ * Firebase Cloud Functions
```
Comment (egilmorez) [inline packages-exp/functions-exp/src/index.ts:2]: The sanctioned name is "Cloud Functions for Firebase."

File: packages-exp/installations-exp/src/api/delete-installations.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ import { FirebaseInstallations } from '../interfaces/public-types';
 
 /**
  * Deletes the Firebase Installation and all associated data.
+ * @param installations - The Installations instance
```
Comment (egilmorez) [inline packages-exp/installations-exp/src/api/delete-installations.ts:27]: Is "Installations" a literal Fei?

If so, we should backtick it, here and throughout (wherever it is a literal).

File: packages-exp/installations-exp/src/api/on-id-change.ts [hunk 3]
```diff
@@ -35,6 +35,9 @@ export type IdChangeUnsubscribeFn = () => void;
 /**
  * Sets a new callback that will get called when Installation ID changes.
  * Returns an unsubscribe function that will remove the callback when called.
+ * @param installations - The Installations instance
+ * @param callback - The callback function which will be invoked when FID changes
```
Comment (egilmorez) [inline packages-exp/installations-exp/src/api/on-id-change.ts:39]: Suggest "callback function that is invoked when the Firebase installation ID changes". 

Globally, avoid "will" unless present tense strictly won't work.

File: packages-exp/installations-exp/src/api/get-id.ts [hunk 4]
```diff
@@ -23,6 +23,7 @@ import { FirebaseInstallations } from '../interfaces/public-types';
 /**
  * Creates a Firebase Installation if there isn't one for the app and
  * returns the Installation ID.
+ * @param installations - The Installations instance
```
Comment (egilmorez) [inline packages-exp/installations-exp/src/api/get-id.ts:26]: Another global nit, and I hate to occupy an engineer's time with this, but -- these and all other non-title strings should end in periods.

See the Firestore tables.  We also added init caps and periods in the FCM strings.  If possible, we should stay consistent with that.  Thanks!

File: packages-exp/performance-exp/src/public_types.ts [hunk 5]
```diff
@@ -42,6 +46,8 @@ export interface FirebasePerformance {
 }
 
 /**
+ * The interface representing a Trace
```
Comment (egilmorez) [inline packages-exp/performance-exp/src/public_types.ts:49]: Is "Trace" a literal?

File: packages-exp/remote-config-exp/src/api.ts [hunk 6]
```diff
@@ -44,7 +44,7 @@ export function getRemoteConfig(app: FirebaseApp = getApp()): RemoteConfig {
 
 /**
  * Makes the last fetched config available to the getters.
- * @param remoteConfig - the remote config instance
+ * @param remoteConfig - The remote config instance.
```
Comment (egilmorez) [inline packages-exp/remote-config-exp/src/api.ts:47]: If we're referring to the product, it should be capped, "Remote Config," here and throughout.

But is it actually a literal, like " `RemoteConfig` instance "  ?

File: packages-exp/remote-config-exp/src/api.ts [hunk 7]
```diff
@@ -185,10 +185,10 @@ export function getNumber(remoteConfig: RemoteConfig, key: string): number {
  * Gets the value for the given key as a String.
  * Convenience method for calling <code>remoteConfig.getValue(key).asString()</code>.
  *
- * @param remoteConfig - the remote config instance
- * @param key - the name of the parameter
+ * @param remoteConfig - The remote config instance.
+ * @param key - The name of the parameter.
  *
- * @returns the value for the given key as a String
+ * @returns The value for the given key as a String.
```
Comment (egilmorez) [inline packages-exp/remote-config-exp/src/api.ts:191]: Does "String" need to be init capped?  If a literal, then backticks please.

File: packages/firestore/exp/index.ts [hunk 8]
```diff
@@ -1,3 +1,9 @@
+/**
+ * Firebase Cloud Firestore
```
Comment (egilmorez) [inline packages/firestore/exp/index.ts:2]: We actually never use "Firebase" for this -- it's just Cloud Firestore.

File: packages/storage/exp/index.ts [hunk 9]
```diff
@@ -1,3 +1,9 @@
+/**
+ * Firebase Cloud Storage
```
Comment (egilmorez) [inline packages/storage/exp/index.ts:2]: Use either "Cloud Storage for Firebase" or just "Cloud Storage"

**Review by egilmorez — COMMENTED** [review]
I have some global nits for you which hopefully won't be too irritating.  Thanks!

File: packages-exp/installations-exp/src/api/delete-installations.ts (hunk 2 above)
Comment (Feiyang1) [reply packages-exp/installations-exp/src/api/delete-installations.ts:27]: It's the product name, like Firebase App

File: packages-exp/performance-exp/src/public_types.ts (hunk 5 above)
Comment (Feiyang1) [reply packages-exp/performance-exp/src/public_types.ts:49]: added backtick

**Review by egilmorez — APPROVED** [review]
Thanks!

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**